common/windows/delphi/ext/cef4delphi/Makefile

```diff
@@ -13,7 +13,7 @@ build: dirs
 
 install:
     $(DEVTOOLS) -ip $(OUTLIB)\CEF4Delphi.bpl
-    $(DEVTOOLS) -ai $(EXT)\cef4delphi\source
+    $(DEVTOOLS) -ai $(KEYMAN_ROOT)\common\windows\delphi\ext\cef4delphi\source
```

Comment:
I take it that we'll worry about new Desktop makefile defines later?  (As we made new ones for Developer toward this purpose earlier in the 🎢 chain.

Reply:
We probably don't need a `$(COMMON_EXT)` define, I think. Is that what you were expecting here?

Reply:
For example, #6719 includes a number of changes that would make good use of that `$(COMMON_EXT)` define - for both platforms.

Reply:
Also, while I missed where in the chain it was introduced, I'd like to point out the existence of `$(COMMON_ROOT)` by the time of #6736.